### PR TITLE
Changed hover effect on disabled delivery methods

### LINKED ISSUE #3547: Hover effect is missing for radio button when hovering on delivery method (text)
**Description**:
Hover effect is missing for radio button when hovering on delivery method (text)

**Expected behavior**:
Hover effect is displayed for radio button as on PDP

**To Reproduce**:
1. Open PDP
2. Add product to cart
3. Open checkout
4. Hover mouse over delivery method (text, NOT radio button)

**Actual result**:
Hover effect is missing for radio button
Screenshot 2021-10-13 at 13 51 57

**Additional context**:
If user hovers mouse over radio button, hover effect is displayed
PDP: https://pr-3529.d25kd40decjpue.amplifyapp.com/default/kids/t-shirts1/bundle-product-with-required-radio-buttons.html

### PR DESCRIPTION
**Pull request by gShubo**
**Related issue(s):**
* Fixes #3547 

**Problem:**
* No hover effect on delivery option text 

**In this PR:**
* Added hover effect on both checkmark  and text 
* Disabled hover effect on disabled delivery options

### CODE REVIEW

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss [hunk 1]
```diff
@@ -51,6 +51,23 @@
         }
     }
 
+    &:hover {
+         &:not(.CheckoutDeliveryOption_isDisabled):not(.CheckoutDeliveryOption_isActive) {
+            .input-control {
+                border-color: var(--checkbox-border-color-active);
+                transition-duration: unset;
+                    &::after {
```
Comment (Sliceem) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:59]: I assume indent not correct
And space before &::after

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss [hunk 2]
```diff
@@ -51,6 +51,23 @@
         }
     }
 
+    &:hover {
+         &:not(.CheckoutDeliveryOption_isDisabled):not(.CheckoutDeliveryOption_isActive) {
```
Comment (Sliceem) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:55]: Can u try to use :not(a, b)

i assume it should work

**Review by Sliceem — CHANGES_REQUESTED** [review]
Good work !
Can u check please comments !

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss [hunk 3]
```diff
@@ -51,6 +51,23 @@
         }
     }
 
+    &:hover {
+         &:not(.CheckoutDeliveryOption_isHoverExcluded){
+            .input-control {
+                border-color: var(--checkbox-border-color-active);
+                transition-duration: unset;
+                &::after {
```
Comment (Sliceem) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:59]: I think u need to add space before &::after

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss [hunk 4]
```diff
@@ -51,6 +51,23 @@
...
+                 @include desktop {
+                --checkmark-color: var(--secondary-dark-color);
```
Comment (Sliceem) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:61]: Indent is wrong

**Review by Sliceem — CHANGES_REQUESTED** [review]
Please see comments !

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss [hunk 5]
```diff
@@ -51,6 +51,24 @@
         }
     }
 
+    &:hover {
+         &:not(.CheckoutDeliveryOption_isHoverExcluded){
```
Comment (Sliceem) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:55]: @AleksandrsKondratjevs  what do you think ???

Comment (AleksandrsKondratjevs) [reply packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:55]: Won't it go with just `&:not(&_isHoverExcluded)` ?

Comment (gShubo) [reply packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:55]: Hey, tried and didn't work

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss [hunk 6]
```diff
@@ -51,6 +51,24 @@
...
+                border-color: var(--checkbox-border-color-active);
+                transition-duration: unset;
+
+                &::after {
+                    @include desktop {
+                        --checkmark-color: var(--secondary-dark-color);
+                    }
+                }
+            }
+            .CheckoutDeliveryOption-Row * {
```
[the remark this answers is not in the document]
Comment (gShubo) [reply packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:66]: sure

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss [hunk 7]
```diff
@@ -51,6 +51,25 @@
...
+                &::after {
+                    @include desktop {
+                        --checkmark-color: var(--secondary-dark-color);
+                    }
+                }
+            }
+            
+            .CheckoutDeliveryOption-Row * {
+                color: var(--primary-base-color);
+            }
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:69]: Split block and element:
```
.CheckoutDeliveryOption {
    &-Row {
         ....
         
```

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss [hunk 8]
```diff
@@ -51,6 +51,27 @@
...
+                &::after {
+                    @include desktop {
+                        --checkmark-color: var(--secondary-dark-color);
+                    }
+                }
+            }
+            
+            .CheckoutDeliveryOption {
+                &-Row * {
+                color: var(--primary-base-color);
```
Comment (AleksandrsKondratjevs) [inline packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss:69]: Indent here is not correct

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**